pyproject.toml

```diff
@@ -13,7 +13,7 @@ classifiers = [
 ]
 dynamic = ["version"]
 
-dependencies = ["typer[all]"]
+dependencies = ["typer"]
```

Comment:
Let's pin it to the latest version to have a more "stable" development, if in the future are new version available then we should update it.

```toml
dependencies = ["typer == 0.12.5"]
```